gym_solo/envs/solo8v2vanilla.py

```diff
@@ -95,15 +108,21 @@ def reset(self, init_call: bool = False) -> solo_types.obs:
       solo_types.obs: The initial observation of the space.
     """
     self.client.removeBody(self.robot)
-    self.robot, _ = self._load_robot()
-
-    # TODO: We need to change this to have the robot always be in home position
-    # Let gravity do it's thing and reset the environment deterministically
-    for i in range(1000):
+    self.robot, joint_cnt = self._load_robot()
+    
+    joint_ordering = [self.client.getJointInfo(self.robot, j)[1].decode('UTF-8')
+                      for j in range(joint_cnt)]
+    positions = [self._config.starting_joint_pos[j] for j in joint_ordering]
+
+    # Let the robot lay down flat, as intended. Note that this is a hack
+    # around modifying the URDF, but that should really be handled in the
+    # solidworks phase
+    for i in range(500):
```

Comment:
Why did you change from 1000 to 500. Also what are these numbers

Reply:
1. Realized that 500 gets the robot in the exact same starting position as 1000, so choosing the 500 cause it's just less time / computation. 

2. What numbers?

Reply:
What does 500 and 1000 signify

Reply:
Just the number of time steps to run to let gravity drop the robot. Basically just a hack to get the robot into a deterministic start position. The number themselves are just magic numbers--they were experimentally found to work.

Reply:
Got it